draft-ietf-httpbis-digest-headers.md

```diff
@@ -263,20 +264,19 @@ It can be used in both request and response.
 The resource is specified by the effective request URI and any `validator`
 contained in the message.
 
-The relationship between Content-Location (see [RFC7231] Section 3.1.4.2)
-and Digest is demonstrated in {{post-not-request-uri}}.
-A comprehensive set of examples showing the impacts of
-representation metadata, payload transformations and HTTP methods on digest
-is provided in {{examples-unsolicited}} and {{examples-solicited}}.
+The relationship between Content-Location (see Section 6.2.5 of
+[I-D.ietf-httpbis-semantics]) and Digest is demonstrated in
+{{post-not-request-uri}}. A comprehensive set of examples showing the impacts of
+representation metadata, payload transformations and HTTP methods on Digest is
+provided in {{examples-unsolicited}} and {{examples-solicited}}.
 
-A Digest header field MAY contain multiple representation-data-digest values.
-This could be useful for responses expected to reside in caches shared by users
-with different browsers, for example.
+A Digest field MAY contain multiple representation-data-digest values. This
+could be useful for responses expected to reside in caches shared by users with
+different browsers, for example.
```

Comment:
```suggestion
A Digest field MAY contain multiple representation-data-digest values.
This could be useful for responses expected to reside in caches shared by users
with different browsers, for example.
```